week-3/C-array-every/exercise.js

```diff
@@ -5,8 +5,12 @@
 var students = ["Omar", "Austine", "Dany", "Swathi", "Lesley", "Rukmini"];
 var group = ["Austine", "Dany", "Swathi", "Daniel"];
 
-var groupIsOnlyStudents; // complete this statement
-
+var groupIsOnlyStudents = group.every(isStudent); // complete this statement
+function isStudent(index) {
+  if (group.includes(students)) {
+    return true;
+  }
```

Comment:
This is technically correct, but usually when we're writing a *predicate* function we want to return either `true` or `false`. Here you are only returning `true` (which is correct!), but you can't returning anything at all if the condition doesn't match.

Reply:
sorry do you mean thats he need to complete it by adding else statement?!

Reply:
correction done to {return group.includes(students);}